app.js

```diff
@@ -0,0 +1,29 @@
+import express from 'express';
+import {healthCheckRoute} from './src/routes/api/health-check.js';
+import {coursesRoute} from './src/routes/api/courses-routes.js'
+
+import swaggerJSDoc from 'swagger-jsdoc';
+import swaggerUi from 'swagger-ui-express';
+
+import { generalErrorHandler } from './src/error-handlers/general-error-handler.js';
+import { notFoundHandler } from './src/error-handlers/not-found-handler.js';
+
+import swaggerOptions from './src/swagger/swagger-options.js';
```

Comment:
Better to rearrange import so external goes first, after imports from local modules.

```suggestion
import express from "express";
import swaggerJSDoc from "swagger-jsdoc";
import swaggerUi from "swagger-ui-express";

import { healthCheckRoute } from "./src/routes/api/health-check.js";
import { coursesRoute } from "./src/routes/api/courses-routes.js";
import { generalErrorHandler } from "./src/error-handlers/general-error-handler.js";
import { notFoundHandler } from "./src/error-handlers/not-found-handler.js";
import swaggerOptions from "./src/swagger/swagger-options.js";
```